GDJS/Runtime/gd.js

```diff
@@ -160,6 +178,23 @@ gdjs.registerBehaviors = function() {
   }
 };
 
+/**
+ * Register the runtime behaviors that can be used bt runtimeObject.<br>
+ *
+ * The type of the behavior must be complete, with the namespace of the extension. For
+ * example, if you are providing a Draggable behavior in the DraggableBehavior extension,
+ * the full name of the type of the behavior is "DraggableBehavior::Draggable".
+ *
+ * @param Ctor The constructor of the Object.
+ * @param behaviorTypeName The name of the type of the behavior.
+ */
+gdjs.registerBehavior = function(Ctor, behaviorTypeName){
```

Comment:
For consistency, I think it's better to  order `register` parameters the same way as the map is: first the key, then what we're registering.
In other words: `function(behaviorTypeName, Ctor)`